plugins/json_processor/src/json_processor/component.py

```
@@ -0,0 +1,228 @@
+                data = self._add_processing_metadata(data)
+
+            final_rows = len(data)
+            logger.info(
+                "Transformation complete: %s -> %s rows",
+                original_rows,
+                final_rows,
+            )
+
+            return Result(status="success", response=data)
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)